src/lib.rs

```diff
@@ -771,6 +771,16 @@ macro_rules! __impl_bitflags {
     };
 }
 
+// Same as std::stringify but callable from __impl_bitflags, which needs to use
+// local_inner_macros so can only directly call macros from this crate.
+#[macro_export]
+#[doc(hidden)]
+macro_rules! __bitflags_stringify {
```

Comment:
Ah interesting... Is this a workaround to an inherent feature of `local_inner_macros` or just an implementation detail that might not be necessary in the future?

Reply:
This is what local_inner_macros means -- macro calls in the expansion are resolved as local helper macros. So a local_inner_macros macro cannot directly call a macro from a different crate, such as stringify from std.